packages/amplify-category-function/resources/awscloudformation/cloudformation-templates/lambda-cloudformation-template.json.ejs

```diff
@@ -214,6 +224,35 @@
         }
         <% } %>
         <% } %>
+        <% if (props.cloudwatchEnabled && props.cloudwatchEnabled != "false") { %> 
```

Comment:
Why is it still props.... and not a CFN condition over the `CloudWatchEnabled` parameter?